ilastik/workflows/newAutocontext/newAutocontextWorkflow.py

```diff
@@ -242,7 +242,7 @@ def handleNewLanesAdded(self):
         # Restore classifier we saved in prepareForNewLane() (if any)
         if self.stored_classifers:
             for pcApplet, classifier in zip(self.pcApplets, self.stored_classifers):
-                pcApplet.topLevelOperator.classifier_cache.forceValue(classifier)
+                pcApplet.topLevelOperator.classifier_cache.forceValue(classifier, set_dirty=False)
```

Comment:
Maybe you could clarify something for me since you've already dove deep into the issue... Is it just the second pixel classification that is retraining? And if so, is it because we're calling `forceValue` on the upstream classifier first, which propagates dirty into the second classifier?

Could this be solved by restoring the classifier cache on the downstream classifier operator first and then on the upstream one?

Reply:
It's the second classifier that is retraining. right now upstream is set first, then downstream. I'd expect that it in fact should only work in this order. The other way round, if you set upstream last, it will propagate dirty downstream completely.

I tried this for fun anyway, and it still retrains :/